deploy/kubernetes/base/daemonset.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: apps/v1beta2 # for versions before 1.8.0 use extensions/v1beta1 [apps/v1beta2]
+kind: DaemonSet
+metadata:
+  name: cadvisor
+spec:
+  template:
+    spec:
+      containers:
+      - name: cadvisor
+        image: gcr.io/google-containers/cadvisor:v0.30.2
+        resources:
+          requests:
+            memory: 200Mi
+            cpu: 150m
+          limits:
+            cpu: 300m
+        volumeMounts:
+        - name: rootfs
+          mountPath: /rootfs
+          readOnly: true
+        - name: var-run
+          mountPath: /var/run
+          readOnly: false
```

Comment:
It would be great if we could make this `readOnly`... do you know why cAdvisor requires this to be writeable?

Reply:
not sure.  I tried running it with `readOnly: true`, and it seemed to work fine